src/Proxy/DriveItemProxy.php

```diff
@@ -9,6 +9,8 @@
 use Microsoft\Graph\Model\Permission;
 use Microsoft\Graph\Model\Thumbnail;
 
+define('CHUNK_SIZE', 60 * 1024 * 1024); //setting chunk size to max level(60 MiB) for upload session
```

Comment:
Can we use `const` instead of `define`?

Also, 60 MiB is quite large for a single PUT request, I would recommend that this be configurable (via `$options`) + validation (eg. should be a multiple of 320 KiB), and set it to a much cheaper default, such as 320 KiB which seems to be the minimum possible.

Reply:
I have this configured via `$options` but, I validated it for max range(i.e if it is within 60MiB or not). Does the chunk size need to be multiple of any number specifically ?